docs/cookbook/keys-descriptors/seed-phrase.md

```diff
@@ -1,38 +0,0 @@
-# From Seed Phrase to Descriptors
```

Comment:
The purpose of this page is to address a common question/need we have seen over and over again on Discord and in private conversation, which is to convert between mnemonics and descriptors.

It is not related to wallet recovery.